src/pages/resources.mdx

```diff
@@ -33,3 +33,5 @@ Watch these talks to learn more about Fastify:
 - [Monoliths always pay their debts - Tomas Della Vedova - NodeJS Italian Conference 2018](https://www.youtube.com/watch?v=6ucyzQULlWw)
 
 - [NodeConf Remote 2020 - Luca Maraschi - From mess to mesh using Fastify](https://www.youtube.com/watch?v=l3mQfvq07CI)
+
+- [Fastify Masterclass - Matteo Collina - 2024](https://www.youtube.com/watch?v=x3SG71Ut2tA&t=1s)
```

Comment:
Not a big deal but is the `t=1s` start time param needed?

Reply:
Well, it saves a second from the viewer. ;)

Reply:
fixed